frontend/src/app.vue

```diff
@@ -32,6 +32,16 @@ export default {
     AppResizePage,
   },
 
+  beforeCreate() {
+    this.$router.beforeEach((to, from, next) => {
+      
+      // TODO: For now we are just attaching static data from each route.
+      // Once translation story implemented we will replace below code with translator function
+      document.title = `crowd.dev${to.meta.title ? ` | ${to.meta.title}` : ''}`;
+      next();
```

Comment:
Just for clarification 😄 What's the translation story this is dependent on?
If there is any link that it can be referred to it would be helpful to have it in the comment

Reply:
#298 is the one i am referring to @joanagmaia. Please feel free to guide me on the same.

Reply:
Yeah for now it's ok to be static because we are not thinking of supporting translations in the near future.
Thanks for removing the comment 🙏